packages/@aws-cdk/aws-iam/test/role.test.ts

```diff
@@ -235,6 +235,49 @@ describe('IAM role', () => {
     });
   });
 
+  describe('role path', () => {
+    test('can be used to specify the path', () => {
+      const stack = new Stack();
+
+      new Role(stack, 'MyRole', { path: '/', assumedBy: new ServicePrincipal('sns.amazonaws.com') });
+
+      expect(stack).toHaveResource('AWS::IAM::Role', {
+        Path: '/',
+      });
+    });
+
+    test('must be between 1 chars and 512 chars', () => {
+      const stack = new Stack();
+
+      const assumedBy = new ServicePrincipal('bla');
+
+      new Role(stack, 'MyRole1', { assumedBy, path: '/' });
+      new Role(stack, 'MyRole2', { assumedBy, path: '/p/a/t/h/' });
```

Comment:
It doesn't look like either of these roles are being used in this test.

Reply:
They are valid cases, When exceptions are raised, the test framework will catch them and make them fail tests. I made it explicit with expect statement in ea3a32439144a1f3597dd0bf14e41f3fe16973b9